stable/mysql/values.yaml

```diff
@@ -4,6 +4,14 @@
 image: "mysql"
 imageTag: "5.7.14"
 
+busybox:
+  image: "busybox"
+  tag: "latest"
```

Comment:
Sorry I missed this one. It should be a specific version, not latest. Bumping to 1.29.3 should be fine